labs/create-lab-flux-config.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+set -ex
+
+NAMESPACE=labs
+PRODUCT=labs
+COMPONENT=$1
+ACR=hmctssandbox
+NAMESPACE_DIR="../apps/${NAMESPACE}"
+COMPONENT_DIR="${NAMESPACE_DIR}/${PRODUCT}-${COMPONENT}"
+ENVIRONMENT="sbox"
+CWD=$PWD
+APPLICATION=${2}
+
+function usage() {
+  echo "Missing an input"
+  echo 'Usage: ./create-lab-flux-config.sh <component> <application>'
+  echo 'Component name - this should be your GitHub username'
+  echo "Application - java or nodejs"
+}
+
+if [ -z "${COMPONENT}" ] || [ -z "${APPLICATION}" ]; then
+  usage
+  exit 1
+fi
+
+# Create component dir and files
+if [ -d "${COMPONENT_DIR}" ]; then
+  echo "Lab directory already exists, run the cleanup script first"
+  exit 1
+else
+  cd ../
+  ./bin/v2/add-image-policies.sh ${NAMESPACE} ${PRODUCT} ${COMPONENT} ${ACR}
+  cd ${CWD}
+fi
+
+if [[ ${APPLICATION} == java ]]; then
+  INGRESS_HOST="${PRODUCT}-${COMPONENT}-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
+elif [[ ${APPLICATION} == nodejs ]]; then
+  INGRESS_HOST="${PRODUCT}-${COMPONENT}-{{ .Values.global.environment }}.platform.hmcts.net"
+else
+  echo "Application type not recognised please use java or nodejs"
+fi
+
+# Create HR for lab
+(
+cat <<EOF
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: labs-${COMPONENT}
+  namespace: labs
+spec:
+  releaseName: labs-${COMPONENT}
+  chart:
+    spec:
+      chart: ./stable/labs-${COMPONENT}
+      sourceRef:
+        kind: GitRepository
+        name: hmcts-charts
+        namespace: flux-system
+      interval: 1m
+  values:
+    ${APPLICATION}:
+      image: hmctssandbox.azurecr.io/labs/${COMPONENT}:latest #{"\$imagepolicy": "flux-system:labs-${COMPONENT}"}
```

Comment:
should have a space so lint doesn't fail I think
```suggestion
      image: hmctssandbox.azurecr.io/labs/${COMPONENT}:latest # {"\$imagepolicy": "flux-system:labs-${COMPONENT}"}
```